setup.py

```diff
@@ -5,12 +5,17 @@
 from setuptools import setup, find_packages
 import sys
 
-# Only 2.7 is supported.
-if not ((2, 7, 0) <= sys.version_info[:3] < (2, 8)):
-    sys.stderr.write('gmusicapi does not officially support this Python version.\n')
-    # try to continue anyway
-
 dynamic_requires = []
+# Python 2.7 is supported. Python 3 support is experimental
+if sys.version_info[0] > 2:
+    sys.stderr.write("gmusicapi Python 3 support is experimental.\n")
+    dynamic_requires.append("protobuf >= 3.0.0b2")
+else:
+    if sys.version_info[:3] <= (2, 7, 0):
```

Comment:
Yes there was (2.7.0 was the check before anyway).

But do we know if all the minor versions back to the initial 2.7 release actually work properly? I know some people on some platforms have had unicode or other odd errors on some older 2.7 minor versions that were seemingly cleared up by installing a newer minor version. Whether that was the actual issue or not is unknown, but it happens.

Perhaps it might be time to tighten it up, since this check just spits out a warning and doesn't actually prevent installation. Maybe 2.7.9 would make sense, since that is the cutoff for the urllb3 InsecurePlatformWarning without the workaround implemented.